src/test/java/com/epam/aidial/cfg/functional/tests/history/ApplicationTypeSchemaHistoryFunctionalTest.java

```diff
@@ -71,6 +72,39 @@ public void shouldSuccessfullyCreateAndUpdateApplicationTypeSchema() {
         Assertions.assertEquals(actualAtOldRevision, applicationsAfterRollback);
     }
 
+    @Test
+    public void shouldSuccessfullyUpdateSchemaWithCorrectHash() {
```

Comment:
why it's in ApplicationTypeSchema`History`FunctionalTest ? it looks more suitable for ApplicationTypeSchemaFunctionalTest